src/oviators/OviatorsExchangeMinterModule.sol

```diff
@@ -138,17 +166,14 @@ contract OviatorsExchangeMinterModule is
     }
 
     function tokenURI(uint256 tokenId) external view returns (string memory) {
-        string memory color = idToColor[tokenId];
-        ColorInfo storage colorInfo = colors[color];
-
         return
             NFTMetadataRenderer.createMetadataEdition({
-                name: string(abi.encodePacked(sink.name(), " ", color)),
+                name: "Oviators", // Token ID is added by NFTMetadataRenderer
                 description: description,
-                imageURI: colorInfo.imageURI,
-                animationURI: colorInfo.animationURI,
+                imageURI: string(abi.encodePacked(imagesBase, tokenId, ".png")),
+                animationURI: string(abi.encodePacked(rendererBase, tokenId)),
                 tokenOfEdition: tokenId,
-                editionSize: maxCount
+                editionSize: 0 // We don't want the NFTMetadataRenderer to name these editioned
```

Comment:
don't fully understand this change, but trusting you 😅

Reply:
if you check `createMetadataJSON` in the `NFTMetadataRenderer`, it adds an extra `tokenId/totalSupply` to the name of each token, in case `editionSize` is anything > 0.

alternative would be to just implement our own metadata JSON renderer vs. using the zora one, but this seems fine too.